src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+
+Some error in `sizes` is acceptable; the attribute provides a pre-layout hint to the browser in order to help it select an appropriate resource to load,_ before_ layout is complete. But large errors can lead to bad resource choices. This appears likely for the least-accurate quarter of `sizes` attributes, which report widths twice as large as the actual `<img>` layout width on desktop and 1.5x as large as the actual `<img>` layout width on mobile.
+
+So: one in ten `sizes` attributes is being authored on the client by a Javascript library, and at least one in four is inaccurate enough that the error is likely to impact resource selection. Both of these facts represent significant opportunities for either <a hreflang="en" href="https://github.com/ausi/respimagelint">existing tooling</a> or <a hreflang="en" href="https://github.com/whatwg/html/issues/4654">new web platform features</a> to help more authors get `sizes` right.
+
+##### `<picture>`
+
+The `<picture>` element serves a couple of use cases:
+
+1. Art direction, with the [`media` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_media_attribute)
+2. Type-switching, via the [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_type_attribute)
+
+{{ figure_markup(
+  caption="The percentage of pages which use `<picture>`.",
+  content="6%",
+  classes="big-number",
+  sheets_gid="620965569",
+  sql_file="picture_distribution.sql"
+)
+}}
+
+`<picture>` is used much less frequently than `srcset`, reflecting its somewhat-niche pair of use-cases. Here's how usage breaks down between those two use cases:
+
+{{ figure_markup(
+  image="picture-feature-usage.png",
+  caption="`<picture>` feature usage.",
+  description="A bar chart showing the percent of pages which use the `media` and `type` attributes on `source` elements, in conjunction with the `picture` element. `media` is used with 48% of `picture` elements on mobile, and just over 44% on desktop. `type` is used with 37% of `picture` elements on mobile, and close to 39% of `picture` elements on desktop.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1613173002&format=interactive",
+  sheets_gid="2031063502",
+  sql_file="picture_switching.sql"
+  )
+}}
+
+Art direction appears a bit more popular than type-switching. Notably, these two features are not mutually exclusive, and, put together, the usage percentages here do not add up to 100%. This suggests that at least 15% of `<picture>` elements do not leverage either of these attributes, making those `<picture>`s  functionally equivalent to a `<span>`.
```

Comment:
That's interesting! Did you look at any examples to see if any ideas WHY they did this?

Reply:
If I had to guess I’d say as apart of a template or as a styling hook, but no, I did not look at specific cases. Worth speculating here (or taking the time to investigate)?